dynamic/Dockerfile

```diff
@@ -1,29 +1,112 @@
+
+ENV LANG=C.UTF-8 \
+    GOMPLATE_VERSION="3.8.0" \
+    GOMPLATE_SHA256="d32817821a7b083b0aeadf29c2aea17e3384b6b619f3a6285150db1914b94c89  v3.8.0.tar.gz"
+
+RUN microdnf install tar gzip wget golang make findutils && \
+    mkdir -p /root/go/src/github.com/hairyhenderson && \
+    cd /root/go/src/github.com/hairyhenderson && \
+    wget https://github.com/hairyhenderson/gomplate/archive/v${GOMPLATE_VERSION}.tar.gz && \
+    echo "${GOMPLATE_SHA256}" | sha256sum --check && \
+    tar xzf v${GOMPLATE_VERSION}.tar.gz && \
+    mv gomplate-${GOMPLATE_VERSION} gomplate && \
+    cd gomplate/ && \
+    GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o gomplate_amd64 ./cmd/gomplate && \
+    GOOS=linux GOARCH=arm64 CGO_ENABLED=0 go build -o gomplate_arm64 ./cmd/gomplate && \
+    GOOS=linux GOARCH=s390x CGO_ENABLED=0 go build -o gomplate_s390x ./cmd/gomplate && \
+    mkdir -p /usr/bin/linux/{amd64,arm64,s390x} && \
+    chmod u+x gomplate_* && \
+    cp gomplate_amd64 /usr/bin/linux/amd64/gomplate && \
+    cp gomplate_arm64 /usr/bin/linux/arm64/gomplate && \
+    cp gomplate_s390x /usr/bin/linux/s390x/gomplate
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS instana-agent
+
+ARG TARGETPLATFORM='linux/amd64'
+ARG DOWNLOAD_KEY
+# This is for backwards compatibility for end users that build their own images
+ARG FTP_PROXY
+
+RUN curl -sSL https://packages.instana.io/Instana.gpg -o /tmp/Instana.gpg && \
+    rpm --import /tmp/Instana.gpg && \
+    export arch=$(case "${TARGETPLATFORM}" in 'linux/amd64') echo 'x86_64' ;; 'linux/arm64') echo 'aarch64' ;; 'linux/s390x') echo 's390x' ;; esac) && \
+    [[ -z "${FTP_PROXY}" ]] || DOWNLOAD_KEY="${FTP_PROXY}" && \
+    echo -e "[instana-agent]\nname=Instana\nbaseurl=https://_:${DOWNLOAD_KEY}@packages.instana.io/agent/generic/${arch}\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://packages.instana.io/Instana.gpg\nsslverify=1" > /etc/yum.repos.d/Instana-Agent.repo && \
+    microdnf install instana-agent-dynamic && \
+    rm -rf /tmp/* /etc/yum.repos.d/Instana-Agent.repo && \
+    microdnf clean all
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+LABEL name="instana-agent" \
+      vendor="Instana Inc." \
+      release="1" \
+      summary="Instana APM agent" \
+      description="Instana APM agent"
+
+ARG TARGETPLATFORM='linux/amd64'
+
+RUN microdnf install --nodocs iptables iproute util-linux python3 jq findutils \
+    && rm -rf /tmp/* \
+    && microdnf clean all
+
+# s390x does not carry the Docker cli
+RUN ( \
+        [[ "${TARGETPLATFORM}" != 'linux/s390x' ]] \
+        && echo -e '[docker-ce-stable]\nname=Docker CE Stable - $basearch\nbaseurl=https://download.docker.com/linux/centos/8/$basearch/stable\nenabled=1\ngpgcheck=1\ngpgkey=https://download.docker.com/linux/centos/gpg' > /etc/yum.repos.d/docker.repo \
+        && microdnf install --nodocs docker-ce-cli tar gzip wget \
```

Comment:
Why is for `static` also `containerd.io` getting installed but not for `dynamic`?
See `static/Dockerfile` line 60
`        && microdnf install --nodocs docker-ce-cli containerd.io \`

Reply:
Good catch!